### PR TITLE
[Term Entry] Lua Math .log()

### PR DESCRIPTION
**Pull request by Fejiro001**
### Description

- Added the .log() method for Lua Math
- Closes Issue #3494    

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/mathematical-library/terms/log/log.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+Title: 'log()'
```
Comment (dakshdeepHERE) [inline content/lua/concepts/mathematical-library/terms/log/log.md:2]:
```suggestion
Title: '.log()'
```

File: content/lua/concepts/mathematical-library/terms/log/log.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Description: 'Calculates the natural logarithm of a number.'
```
Comment (dakshdeepHERE) [inline content/lua/concepts/mathematical-library/terms/log/log.md:3]:
```suggestion
Description: 'Calculates the natural logarithmic value of a number.'
```

File: content/lua/concepts/mathematical-library/terms/log/log.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - 'Game Development'
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`log()`** function in Lua's math library calculates the natural logarithm of a given number. The natural logarithm is to the base `e`, where `e` is Euler's number, an irrational constant approximately equal to 2.71828.
```
Comment (dakshdeepHERE) [inline content/lua/concepts/mathematical-library/terms/log/log.md:15]:
```suggestion
The **`.log()`** function in Lua's math library calculates the natural logarithm of a given number. The natural logarithm is to the base `e`, where `e` is Euler's number, an irrational constant approximately equal to `2.71828`.
```

File: content/lua/concepts/mathematical-library/terms/log/log.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+```pseudo
+math.log(number)
+```
+
+- `number`: The number for which the natural logarithm is calculated.
+
+## Example
+
+In this example, `math.log()` calculates the natural logarithm of 20.
```
Comment (dakshdeepHERE) [inline content/lua/concepts/mathematical-library/terms/log/log.md:27]:
```suggestion
In this example, `math.log()` calculates the natural logarithmic value of `20`:
```

File: content/lua/concepts/mathematical-library/terms/log/log.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+```lua
+local num = 20
+local result = math.log(num)
+
+local output = string.format("The natural Logarithm of %.0f: %.4f", num, result)
+print(output)
+```
+
+The result of the above example is shown below.
```
Comment (dakshdeepHERE) [inline content/lua/concepts/mathematical-library/terms/log/log.md:37]:
```suggestion
The above example will result in the following output:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have requested few changes for you @Fejiro001 🚀

**Comment by dakshdeepHERE** [comment]
Hey @Fejiro001, Thank you for contributing this entry. This looks good for second review.

**Review by dakshdeepHERE — APPROVED** [review]

File: content/lua/concepts/mathematical-library/terms/log/log.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+## Example
+
+In this example, `math.log()` calculates the natural logarithmic value of `20`:
+
+```lua
+local num = 20
+local result = math.log(num)
+
+local output = string.format("The natural Logarithm of %.0f: %.4f", num, result)
+print(output)
```
Comment (avdhoottt) [inline content/lua/concepts/mathematical-library/terms/log/log.md:34]:
```suggestion
local num = 20
local result = math.log(num)
local output = string.format("The natural Logarithm of %.0f: %.4f", num, result)
print(output)
```
Remove the extra line from the code.

**Review by avdhoottt — APPROVED** [review]
Looks good to merge!!

**Pull request merged**